app/views/admin/enterprises/form/_users.html.haml

```diff
@@ -17,27 +10,25 @@
       %a= t('admin.whats_this')
   .eight.columns.omega
     - if full_permissions
-      = f.hidden_field :owner_id, class: "select2 fullwidth", 'user-select' => 'Enterprise.owner'
+      = f.hidden_field :owner_id, class: "select2 fullwidth", 'user-select' => 'Enterprise.owner', 'ng-model' => 'Enterprise.owner'
```

Comment:
I am 90% this is just overwriting `Enterprise.owner` (a representation of the User object) with an integer (owner_id) that is never used. It probably harmless, but I don't think this is what we want to do.

Reply:
I think I added `ng-model` here because the owner id is being used by angular elsewhere on this page, and it needed to be dynamically updated when the other form elements change.

Reply:
Ok, but just remember that `ng-model` works both ways, so when someone uses this select, it's going to overwrite the value of `Enterprise.owner` with whatever the selected value is. I thought it was an integer, but perhaps it's not. If it is an integer, will that have consequences in other parts of a page that are expecting `Enterprise.owner` to be an object? If it's not an integer, I have no idea how submitting this as a rails form is working.

Reply:
@oeoeaio This element uses the `user-select` directive, which queries the database for users based on input in the select2 search, then binds the returned user object (including the id) onto the specified `ng-model`, if one was passed to the directive. It's not overwriting it with just an id.

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/javascripts/admin/users/directives/user_select.js.coffee